### DISCUSSION TITLE
Timeline: missing formatTimeCallback

### DISCUSSION THREAD

**Discussion by TS-Stefan** [katspaugh/wavesurfer.js#2985] [Q&A] [answered]
Hi,

with v7 the formatTimeCallback has gone from the timeline plugin, which was useful to format the time other than default.
Was it removed intentionally? If so, what was the reason?

Thanks in advance

**Comment by katspaugh** [comment, marked as answer]
It wasn't intentionally removed, I just didn't port it. So good catch!
Please feel free to restore it, the old code is [here](https://github.com/katspaugh/wavesurfer.js/blob/v6/src/plugin/timeline/index.js#L531-L548). Should be pretty easy to copy it.

**Reply by TS-Stefan** [reply]
hope this is ok:
https://github.com/katspaugh/wavesurfer.js/pull/2986/files

**Reply by katspaugh** [reply]
Thank you!
